core/src/Platforms/Android/SystemWebview/AuthenticationActivity.cs

```diff
@@ -61,7 +60,7 @@ public AuthenticationActivity()
         private string _requestUrl;
         private int _requestId;
         private bool _restarted;
-       // private CustomTabsActivityManager _customTabsActivityManager;
+        // private CustomTabsActivityManager _customTabsActivityManager;
```

Comment:
you might want to remove this comment since it has been here already?

Reply:
done...thanks!